### DISCUSSION TITLE
Autostart

### DISCUSSION THREAD

**Discussion by Joergold** [Waboodoo/HTTP-Shortcuts#444] [Q&A]
It would be nice to have an examlpe to trigger HPPT-S by sytem clock.
Can annyone help?

**Comment by Waboodoo** [comment]
The app itself currently does not have a feature to create clock-based triggers like that. But you could achieve this by using a dedicated automation app like Tasker or Macrodroid. These apps let you create scenarios/workflows like that, which can then trigger a shortcut in the HTTP Shortcuts app.
